client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -60,6 +61,15 @@ export class CommentNavigation extends Component {
 
 	shouldComponentUpdate = nextProps => ! isEqual( this.props, nextProps );
 
+	componentDidUpdate = prevProps => {
+		const { commentsListQuery, hasPendingBulkAction, refreshPage } = this.props;
+		if ( ! hasPendingBulkAction && prevProps.hasPendingBulkAction ) {
+			if ( commentsListQuery ) {
+				refreshPage( commentsListQuery );
+			}
+		}
+	};
```

Comment:
is there a reason for putting this here instead of in `componentWillReceiveProps()`?

Reply:
To my knowledge we shouldn't perform side effects in compoentWillReceiveProps, see https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9#gistcomment-2280980

Reply:
wow I had no idea this was the case